packages/payments-service/src/core/server.ts

```diff
@@ -1,34 +1,52 @@
-import express, { RequestHandler } from 'express'
-import bodyParser from 'body-parser'
-import session from 'express-session'
-import memoryStore from 'memorystore'
-import { v4 as uuid } from 'uuid'
-import cors from 'cors'
-import { traceIdMiddleware } from '@reapit/utils-node'
-import router from './router'
-const MemoryStore = memoryStore(session as any)
-
-const app = express()
-
-app.disable('x-powered-by')
-app.use(cors())
-
-app.use(
-  session({
-    secret: uuid(),
-    resave: false,
-    saveUninitialized: true,
-    store: new MemoryStore({
-      checkPeriod: 86400000, // prune expired entries every 24h
-    }),
-  }),
-)
-
-app.use(traceIdMiddleware)
-app.use(bodyParser.json() as RequestHandler)
-app.get('/ok', (_req, res) => {
-  res.status(200).send('ok').end()
-})
-app.use(router)
-
-export default app
+import { INestApplication, Logger, ValidationPipe } from '@nestjs/common'
+import { NestFactory } from '@nestjs/core'
+import { createServer, proxy } from 'aws-serverless-express'
+import express, { Express } from 'express'
+import { ExpressAdapter } from '@nestjs/platform-express'
+import { eventContext } from 'aws-serverless-express/middleware'
+import { Server } from 'http'
+import { Handler, APIGatewayEvent, Context } from 'aws-lambda'
+import { CorsHeaderInterceptor } from '@reapit/utils-nest'
+import { AppModule } from './app'
+import config from '../../config.json'
+
+export const bootstrapApplication = async (): Promise<[INestApplication, Express]> => {
+  process.env = {
+    ...process.env,
+    ...config,
+  }
+
+  const expressApp = express()
+  const app = await NestFactory.create(AppModule, new ExpressAdapter(expressApp))
+
+  app.useGlobalPipes(new ValidationPipe())
```

Comment:
This is where the validationPipe comes from, in future if you want to use `class-validator` you'll need to add this line to your project.

Reply:
Got you 👍🏻